src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -67,7 +70,7 @@ def generate_result_report(self):
             "job_id": self.job_id,
```

Comment:
Since we are already in result enhancement domain, good time to deal with `D3ASIM-2288` after this ticket to reduce some more memory issues via replacing pendulum timestamp to UNIX.

Reply:
I am definitely a fan of D3ASIM-2288, but it needs to be discussed further, in the frame of the discussion of the simulation results service.